vendors/st/boards/stm32l475_discovery/CMakeLists.txt

```diff
@@ -155,6 +155,15 @@ target_sources(
         $<${AFR_IS_TESTING}:${afr_ports_dir}/common_io/iot_test_common_io_internal.c>
 )
 
+target_include_directories(
```

Comment:
Did Espressif need an update as well? Or does it still compile

Reply:
Espressif does compile as validated both locally and from PR build checks